### PR TITLE
add the Generate and Publish Workflow Report step

### PR DESCRIPTION
**Pull request by malacourse**

### CODE REVIEW

**Comment by itewk** [comment]
@malacourse until we have a first step implementer in the ploigos-step-runner project for this step we shoudl hold off on merging it because it will error.

File: vars/ploigosWorkflowMinimal.groovy [hunk 1]
```diff
@@ -522,6 +522,21 @@ def call(Map paramsMap) {
                             }
                         }
                     }
+                    stage('CI: Generate Meta Data Report') {
```
Comment (itewk) [inline vars/ploigosWorkflowMinimal.groovy:525]: please update name to `Report: Generate and Publish Workflow Report`

**Comment by malacourse** [comment]
Did not mean to apply to this repo

**Comment by itewk** [comment]
@malacourse all good.

**Pull request closed without merging**